src/content/configuration/other-options.md

```diff
@@ -100,6 +100,8 @@ Expose custom values into the loader context.
 
 Limit the number of parallel processed modules. Can be used to fine tune performance or to get more reliable profiling results.
 
+default: 100
```

Comment:
Would you mind moving this to previous paragraph and re-word to `Defaults to '100'.` Thanks!